### ISSUE TITLE
No Seed despite "correct" data

### ISSUE THREAD

**Issue by coldReactive** [Treeki/MeteoNook#21]
A couple hourly slots I got data around the 30-50 minute mark, so its data isn't exactly accurate:

https://pastebin.com/Ku4CgbLR

**Comment by gjnance** [comment]
I have had a similar issue. I repeated the information gathering process multiple times personally, using the vaguest options (Rain/No Rain), but two days of meticulously gathered shooting stars. Currently we have a seed, but adding new data to that seed will often negate it (particularly an accurate shooting star time).

Something interesting is happening, however. The prediction of shooting stars for our island seems to fall on the correct day, but the times do not match up. I find this pretty odd considering the precise timings the app generates.

I feel like something is a bit off with the core algorithm here.

**Comment by Starwort** [comment]
Personally, I've found the seed for 5 islands so I suspect there's something slightly off with your data (the first time I did it, I accidentally reported a meteor shower on the wrong day!) - make sure anything 12am - 5am is on the calendar day before the real day. Alternatively, if you can get times to the second accurately then one meteor shower should be enough; finding your seed with minimal data is the best way (and narrowing down your last few seeds manually based on meteor showers is the best way to ensure that they are correct)

**Comment by deiussum** [comment]
I've been running into issues tracking down a seed as well.  My wife has screenshots of somewhere around 20 meteors and a rainbow.  Without even trying to use the seconds, she gets no seed found.  I've scanned over the screenshots and data several times and haven't been able to find anything entered wrong.  I've made sure anything after midnight is registered on the day before the real calendar day.  

On my own island, I've got around 12-15 meteor screenshots.  If I try to use the seconds, no seed found.  Without the seconds, a couple of rain showers, and a couple of gaps where I know there were no meteors, I'm down to 3 and now seeing if I can manually narrow that down based on some upcoming rainbow/meteor events in those seeds.

Might try resetting my data and entering everything again a few entries at a time.

**Comment by Starwort** [comment]
> Might try resetting my data and entering everything again a few entries at a time.

This is what I did when trying to figure out why I wasn't getting any seeds (and how I figured out I messed up my shower date). Also avoid pressing 'none of these' or whatever the label is as you may have missed a light shower

**Comment by deiussum** [comment]
FYI, I think I found the seed for my island.  After comparing the meteor seconds to the seconds on my screenshots I found that my screenshots were all off by 2-4 seconds, with one of them being off by 5 seconds.  Since the threshold for seconds is only 4 seconds, that would explain why entering all my seconds eliminated this seed.  My guess is I was just a bit late on capturing the screenshot on that one.  

My wife had a couple that were real close to the change of the minute, so it is possible that even though the in-game clock showed one time, the star may have started a second or two before that minute.  Armed with this knowledge, I plan to go through her screenshots a bit more carefully and subtract a couple seconds from any stars she caught near the end of their streak.

Thought I would share in case this helps anyone else out.

**Comment by aricagiftedrn** [comment]
I have entered a rainbow and a light shower to the minutes over 2 hours and it tells me no seed found. The data is 100% accurate. There might be an issue with shooting star times.

**Comment by gjnance** [comment]
@Treeki - https://photos.app.goo.gl/vPPSKieptg5EV4F68

If the 9am weather is correct as "Unsure Sunny/Cloudy", then the 12:04 weather must be clear (Fine04 or Fine06), yet there are obviously clouds on the horizon.

Is it possible there is more nuance to weather types than the still images on the seed page indicate?

**Comment by coldReactive** [comment]
Yeah, I just had those cumulonimbus clouds on the horizon a couple days ago during the evening / night. I don't know what they signified, as the next day was clear/fine/sunny all day.

**Comment by gjnance** [comment]
Might be nice to add a hover for the weather images that pops up a panel with 3 or 4 varying examples that span the range for that weather type.

**Comment by Treeki** [comment]
The special cloud types like cumulonimbus are independent from the "main" weather types (clear/sunny/cloudy), and only show up during certain parts of the year.

I haven't added info about them to MeteoNook yet because I need to do some more research on exactly which combinations of patterns trigger them and confirm which times of day they're visible at.

**Comment by myrrien** [comment]
It's the fourth time now I try to get my seed, everytime I do it from scratch, at this point I'm only inputting 'clear' and 'unsure' as weathers since everytihng else seems to invalidate my seed search
